examples/bridge-app/linux/main.cpp

```diff
@@ -989,7 +989,9 @@ int main(int argc, char * argv[])
 
     // Setup composed device with two temperature sensors and a power source
     ComposedDevice ComposedDevice("Composed Device", "Bedroom");
-    DevicePowerSource ComposedPowerSource("Composed Power Source", "Bedroom", EMBER_AF_POWER_SOURCE_FEATURE_BATTERY);
+    DevicePowerSource ComposedPowerSource(
+        "Composed Power Source", "Bedroom",
+        chip::BitFlags<PowerSource::PowerSourceFeature>(PowerSource::PowerSourceFeature::kBattery));
```

Comment:
```suggestion
        PowerSource::PowerSourceFeature::kBattery);
```
might compile too...